manticore/platforms/linux.py

```diff
@@ -320,9 +308,9 @@ def _init_fds(self):
         self.input.peer = stdin
         #A receive on stdout or stderr will return no data (rx_bytes: 0)
 
-        assert self._open(stdin) == 0
```

Comment:
I don't mind. It had taken me a while to get https://github.com/trailofbits/manticore/blob/a1a2dfd3ec68294c4a2a84b1b026bc80d3b84de6/manticore/platforms/linux.py#L1615, so I figured if we are to use STDIN, STDOUT etc better to name them somehow, thus I used sys.__stdin__.fileno() so as to not define a new var. I overall avoid any sort of number which is not 0, 1, -1, so as to give more context, but no hard feelings on this